python/lsst/rubintv/production/allSky.py

```diff
@@ -104,7 +108,70 @@ def dayObsFromDirName(fullDirName, logger):
         return None, None
 
 
-def _convertJpgScale(inFilename, outFilename):
+def getUbuntuFontPath(logger=None):
+    """Get the path to the Ubuntu font, if available.
+
+    Parameters
+    ----------
+    logger : `logging.logger`
+        The logger.
+
+    Returns
+    -------
+    ubuntuBoldPath : `str`
+        The path to the Ubuntu bold font, or ``None`` if not found.
+    """
+    fontPaths = matplotlib.font_manager.findSystemFonts(fontpaths=None, fontext='ttf')
+
+    ubuntuBoldPath = [f for f in fontPaths if f.find('Ubuntu-B.') != -1]
+    if not ubuntuBoldPath:
+        if not logger:  # only create if needed
+            logger = _LOG.getChild("getUbuntuFontPath")
+        logger.warning('Warning - cound not fund Ubuntu bold font!')
+        return
```

Comment:
In this case, will the annotation code simply use a default system font?

Reply:
For better and for worse - yes. It's different on every system I've tried it on, and sometimes it's awful, but this way failover is just a bad font, not failure, so it seemed like the best way to go.